.gitmodules

```diff
@@ -1,4 +1,5 @@
+
 [submodule "libdedx"]
 	path = libdedx
 	url = https://github.com/APTG/libdedx.git
-	shallow = true
+	branch = change_get_functions
```

Comment:
Why `shallow` mode was removed ?

Reply:
My bad, during the process i removed the whole submodule and added it back - i forgot to include this